src/components/Loading Screen/LoadingScreen.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components/macro'
+import BounceLoader from 'react-spinners/BounceLoader'
+
+export default function LoadingScreen() {
```

Comment:
For me, a screen is similar to a page, but this component is used inside a page. Maybe think about a different name.
e.g.
```suggestion
export default function LoadingSpinner() {
```

Reply:
Yes that is a fair point and i changed my function name accordingly.